staking-pool/README.md

```diff
@@ -400,10 +400,6 @@ near login
 ```bash
 near call my_validator deposit '{}' --accountId user1 --amount 100
 ```
-#### Stake the deposited amount to the validator contract
```

Comment:
Why did you remove this? It seems useful. When you just deposit, the amount is not staked immediately

Reply:
Bowen said the contract automatically stakes for you so you should not do it directly. Sorry for the confusion. I will fix that and get your feedback integrated. Only thing missing is creating the new node with the contract account. I will add it.